roles/benchmarking_run_nvidiadl_ssd/tasks/main.yml

```diff
@@ -77,3 +77,15 @@
       oc describe pod/{{ benchmarking_nvidiadl_ssd_name }} -n {{ benchmarking_namespace }}
         > "{{ artifact_extra_logs_dir }}/pod_run-nvidiadl-ssd.descr"
     failed_when: false
+
+- name: Get average sample rate
+  shell:
+    set -o pipefail;
+    cat "{{ artifact_extra_logs_dir }}/pod_run-nvidiadl-ssd.log" | grep avg. | tail -n1 | awk '{ print $NF }' > "{{ artifact_dir }}/sample_rate.log"
```

Comment:
🤔 
I wasn't thinking about it this way, but it might be a good idea
I was thinking: write it into `{{ artifact_extra_logs_dir }}`, and fetch it from there in the calling script, but it makes sense like that as well.

Could you 
1/ prefix the file name with the role: `"{{ artifact_dir }}/benchmarking_run_ssd_sample_rate.log"`
2/ write the file as well in `{{ artifact_extra_logs_dir }}`

for both files of course